setup.py

```diff
@@ -10,7 +10,7 @@ def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 
-install_requires_list = []
+install_requires_list = ["twine>=1.7"]
```

Comment:
do we need to remove a reference to distutils anywhere? or was that just installed by default...?

Reply:
I don't see references to it in this repo so I think it was installed by default

Reply:
I came to the same conclusion. Aight, ship it then! Once you get that build error ironed out